course3/homework-unique_ptr/unique_ptr.h

```diff
@@ -0,0 +1,70 @@
+#pragma once
+
+#include <cstddef>
+#include <utility>
+
+
+namespace karun
+{
+	template <typename T>
```

Comment:
There's no point indenting the entire namepsace. It only wastes lot of screen space, and adds no readability.